src/api/cow/cow.spec.ts

```diff
@@ -0,0 +1,441 @@
+import { ethers } from 'ethers'
+import fetchMock, { enableFetchMocks } from 'jest-fetch-mock'
+import { CowSdk } from '../../CowSdk'
+import { OrderKind, SigningScheme } from '@gnosis.pm/gp-v2-contracts'
+import OperatorError from './errors/OperatorError'
+import { PriceQuoteParams } from '../../utils/price'
+import QuoteError from './errors/QuoteError'
+import { CowError } from '../../utils/common'
+import { OrderCreation, UnsignedOrder } from '../../utils/sign'
+
+enableFetchMocks()
+
+const chainId = 4 //Rinkeby
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const gpV2Contracts = require('@gnosis.pm/gp-v2-contracts')
+
+// @ts-expect-error: Let's ignore a compile error like this window.ethereum undefined
+const provider = new ethers.providers.Web3Provider(window.ethereum)
+
+const cowSdk = new CowSdk(chainId, { signer: provider.getSigner() })
+
+const HTTP_STATUS_OK = 200
+const HTTP_STATUS_NOT_FOUND = 404
+
+const ORDER_RESPONSE = {
+  sellToken: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+  buyToken: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+  receiver: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+  sellAmount: '1234567890',
+  buyAmount: '1234567890',
+  validTo: 0,
+  appData: '0x0000000000000000000000000000000000000000000000000000000000000000',
+  feeAmount: '1234567890',
+  kind: 'buy',
+  partiallyFillable: true,
+  sellTokenBalance: 'erc20',
+  buyTokenBalance: 'erc20',
+  signingScheme: 'eip712',
+  signature:
+    '0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
+  from: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+  creationTime: '2020-12-03T18:35:18.814523Z',
+  owner: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+  uid: 'string',
```

Comment:
I know this is irrelevant for this test, but could you use a fake string in the format expected, just for the sake of being a somewhat valid response?